src/Orleans.Core.Abstractions/Runtime/GrainReference.cs

```diff
@@ -1,11 +1,27 @@
 using System;
+using System.Diagnostics;
+using System.Diagnostics.Tracing;
 using System.Runtime.Serialization;
 using System.Threading.Tasks;
 using Orleans.CodeGeneration;
 using Orleans.Serialization;
 
 namespace Orleans.Runtime
 {
+    public class OrleansGrainReferenceEvent : EventSource
```

Comment:
We should specify the name explicitly & attribute the event-emitting methods, too

Reply:
what do you mean? The name will be OrleansGrainReferenceEvent  by default, is that not enough?

Reply:
I think the convention for event source names is more like "Microsott-Orleans-GrainReference" or something like that